.github/workflows/release-tag-and-build.yml

```
@@ -32,18 +26,32 @@ jobs:
           tar czf ott-web-app-build-${RELEASE_VERSION}.tar.gz public
           zip -r ott-web-app-build-${RELEASE_VERSION}.zip public
 
+      - name: Get current package version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
```

We can probably use `jq` to parse the file:

```suggestion
      - name: Get current package version
        id: package-version
        run: |
            echo "PACKAGE_JSON=$(jq -c . < package.json)" >> $GITHUB_ENV
        - run: |
            echo '${{ fromJson(env.PACKAGE_JSON).version }}'
```

And then use

---

Good point, I guess we don't need a dependency for this